SYCL/KernelAndProgram/build-log.cpp

```diff
@@ -1,8 +1,6 @@
-// XFAIL: cuda
 // RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple  %s -o %t.out
-// RUN: %CPU_RUN_PLACEHOLDER %t.out
 // RUN: %GPU_RUN_PLACEHOLDER %t.out
-// RUN: %ACC_RUN_PLACEHOLDER %t.out
+// REQUIRES: gpu
```

Comment:
Wouldn't it be useful to write here a comment telling why this test requires gpu?
It also seems like there is no intention to run this test on CPU/ACC in future, right?